src/productcatalogservice/catalog_loader.go

```diff
@@ -74,13 +75,16 @@ func getSecretPayload(project, secret, version string) (string, error) {
 	// Call the API.
 	result, err := client.AccessSecretVersion(ctx, req)
 	if err != nil {
+		log.Warnf("failed to access SecretVersion: %v", err)
```

Comment:
Warn logs are a great addition here

Reply:
I was trying to troubleshoot what was happening with the products catalog, and figured I'd leave them for the next time there's a potential issue :)